typescript-getting-started/functions/test/test.ts

```diff
@@ -0,0 +1,21 @@
+import { suite, test } from '@testdeck/mocha'
+import * as chai from 'chai'
+import * as functions from 'firebase-functions'
+
+import { helloWorld } from '../src'
+
+const assert = chai.assert;
+
+@suite
+class HelloWorld {
+  @test
+  async helloWorld() {
+    const req = {} as functions.https.Request;
```

Comment:
used `as`.
It's because I don't add mock library for only this and this test is very small and simple.